seed/challenge-style-guide.md

```diff
@@ -63,6 +63,38 @@ Here are some example challenge names:
 - Condense arrays with .reduce
 - Use Bracket Notation to Find the First Character in a String
 
+## Numbering Challenges
+
+Every challenge needs an `id`. If you don't specify one, then MongoDB will create a new random one when it saves the data; however, we don't want it to do that, since we want the challenge ids to be consistent across different environments (staging, production, lots of different developers, etc.).
+
+To generate a new one in a shell (assuming MongoDB is running separately):
+
+1. Run `mongo` command
+2. Run `ObjectId()` command
+
+For example:
+
+```sh
+$ mongo
+MongoDB shell version v3.6.1
+connecting to: mongodb://127.0.0.1:27017
+MongoDB server version: 3.4.10
+...
+$ ObjectId()
+ObjectId("5a474d78df58bafeb3535d34")
+```
+
+The result is a new id, for example `5a474d78df58bafeb3535d34` above.
+
+Once you have your id, put it into the JSON file as the `id` field, e.g.
+
+```json
+{
+  "id": "5a474d78df58bafeb3535d34",
+  "title": "Challenge Title",
+```
```

Comment:
...and this gets weird again. GitHub's diff algorithm is apparently unaware that the triple-tick closes the JSON block and marking it and the next few lines red. Looking at the entire file shows there is no problem with their actual markdown formatter: 
![image](https://user-images.githubusercontent.com/8524/35163715-aeb19590-fd15-11e7-8bd1-3fb608d350b9.png)

Reply:
I agree, I think adding the `json` is doing nothing special with regards to highlight other making it unreadable in the future diffs, lets just remove it.

Reply:
Other than that everything looks good for merge.